Tests/MongoSwiftSyncTests/CrudV2Tests.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import MongoSwiftSync
+import Nimble
+import TestsCommon
+import XCTest
+
+// TODO: remove with SWIFT-780
+/// A place for CrudV2 Tests until the swift crud v2 runner is shipped
+final class MongoCrudV2Tests: MongoSwiftTestCase {
+    func testFindOptionsAllowDiskUse() throws {
+        let client = try MongoClient.makeTestClient()
+        let monitor = client.addCommandMonitor()
+
+        try self.withTestNamespace { _, _, coll in
+            try coll.insertOne(["dog": "notCat"])
+
+            try monitor.captureEvents {
+                let optionAllowDiskUseNil = FindOptions()
+                expect(try coll.find(["dog": "notCat"], options: optionAllowDiskUseNil)).toNot(throwError())
+
+                let optionAllowDiskUseFalse = FindOptions(allowDiskUse: false)
+                expect(try coll.find(["dog": "notCat"], options: optionAllowDiskUseFalse)).toNot(throwError())
+
+                let optionAllowDiskUseTrue = FindOptions(allowDiskUse: true)
+                expect(try coll.find(["dog": "notCat"], options: optionAllowDiskUseTrue)).toNot(throwError())
+            }
+
+            let events = monitor.commandStartedEvents()
+            expect(events).to(haveCount(3))
+
+            let eventAllowDiskUseNil = events[0]
+            expect(eventAllowDiskUseNil).toNot(beNil())
```

Comment:
I think we can get rid of the the assertions that the events are non-nil. the return type of `commandStartedEvents()` is an array of non-optionals, so it's impossible that we ever end up with nil values for these variables.